Header header

geometry_msgs/Vector3 moment
float64 total_thrust
